validation_pipeline.ipynb

```diff
@@ -25,12 +25,14 @@
         "DEBUG = False #@param {type:\"boolean\"}\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    TEST_SET_FILENAME = 'canonical/uc_davis_test_fixed_grouped.csv' #@param
</pre><p>What does "grouped" mean in this context? There was some notion of "grouped" and "ungrouped" as well as per-sample and per-site. How does that fit in here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
It means that it's the test dataset we got from data_ingestion.ipynb by grouping by Code, lat, long.